lib/router/transition.js

```diff
@@ -107,6 +115,8 @@ Transition.prototype = {
   promise: null,
 
   /**
+    @property data
```

Comment:
Same as above, these yuidoc statements need to be moved down below the prose description.